microservices/payment/src/entities/transaction.ts

```diff
@@ -1,6 +1,74 @@
-import { Entity } from 'typeorm';
+import { IsUndefinable } from '@lomray/microservice-helpers';
+import { Allow } from 'class-validator';
+import { JSONSchema } from 'class-validator-jsonschema';
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryColumn } from 'typeorm';
+import Customer from '@entities/customer';
 
+@JSONSchema({
+  properties: {
+    customer: { $ref: '#/definitions/Customer' },
+  },
+})
 @Entity()
-class Transaction {}
+class Transaction {
+  @PrimaryColumn({ type: 'varchar', length: 66 })
+  @Allow()
+  id: string;
```

Comment:
why it not auto increment? uuid? (if it's external transaction id, you should store it in separated field

Reply:
yes, this field is external. The first version of this field I named trnasactionId. Is it correct?